spec/spec_helper.rb

```diff
@@ -1 +1,23 @@
+if ENV['COVERAGE']
```

Comment:
could all these requires be wrapped into a method and called in setup-simplecov before `require "simplecov"`?